px-app-route/px-app-route-helpers.js

```diff
@@ -0,0 +1,113 @@
+(function() {
+  var PxAppRouteHelpers = {
+    isDeepEqual: function(a1, a2) {
+      if (!Array.isArray(a1) || !Array.isArray(a2)) return false;
+      if (a1.length !== a2.length) return false;
+
+      for (let i=0; i<a1.length; i++) {
```

Comment:
👍  good find, just fell back to var for now so we don't have to worry about transpilation for this.